common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -739,6 +787,21 @@ def get_transcripts_info(self, is_bumper=False):
             language_code: transcript_file
             for language_code, transcript_file in transcripts.items() if transcript_file != ''
         }
+
+        # For phase 2, removing `include_val_transcripts` will make edx-val
+        # taking over the control for transcripts.
+        if include_val_transcripts:
```

Comment:
This will be executed for bumper  as well, are we sure we need this for bumper too ?

Reply:
We still need to be decide this -- include or not.